pkg/controller/pxc/controller.go

```diff
@@ -353,6 +353,22 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(request reconcile.Request) (re
 		}
 	}
 
+	if len(o.Spec.PXC.ReplicationChannels) != 0 {
+		err = r.ensurePxcPodServices(o)
```

Comment:
I think we should create service-per-pod only if at least one Replication channel exists with `isSource: true`